test/test.js

```diff
@@ -0,0 +1,84 @@
+var nock   = require('nock');
+
+var logic = require('../logic.js');
+
+function responseLog(response) {
+  console.log('   RESPONSE SENT: ' +  response);
+}
+
+var ghUser = nock('https://api.github.com')
+              .get('/user')
+              .reply(200, {
+                login: 'testuser',
+                id: 1234567,
```

Comment:
you can put only the parameters we need. and maybe some important few other parameters. like id and avatar_url.